### ISSUE TITLE
How can I run banpick

### ISSUE THREAD

**Issue by Kuro2706** [floh22/LeagueBroadcastHub#257]
I downloaded it and couldn't run pick ban. I read the instruction but still don't know about that. Can you help me please

**Comment by daisukedd** [comment]
add me on discord daisuke#0132

**Comment by Kuro2706** [comment]
I have sent a friend request on Discord

**Comment by shinilee** [comment]
yup to me it doesn't work too it used to work but it dosn't work anymore I use tournament code to create the room for 10 people and it doesn't appear when the ban pick start so I'm just using the program without ban pick.

**Comment by Kuro2706** [comment]
hope someone can make a tutorial video for all features.

**Comment by FireFoXz09** [comment]
I always had issues with the integrated pick&ban feature of LeagueBroadcast (working 1/3 time) but a friend of mine made it work EVERY TIME (and we don't know why it works on his side and not mine.) So I'm using the stand alone version from RCVolus here: https://github.com/RCVolus/lol-pick-ban-ui . (It's the program used by LeagueBroadcast)
